docs/main.mjs

```diff
@@ -23,3 +29,37 @@ document.querySelectorAll('.copy-icon').forEach(icon => {
         }, 750)
     })
 })
+
+backToTopBtn.addEventListener('click', (e) => {
```

Comment:
Let's avoid adding lot's of things to this file. Create a new module file in the [modules](https://github.com/animate-css/animate.css/tree/main/docs/modules) folder and wrap it all in an exported function which will get imported and executed in this file. See the other files in this folder for examples.